### PR TITLE
ISPN-12506 Fix protobuf schema lack of permission

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-12506

### CODE REVIEW

File: src/services/protobufService.ts [hunk 1]
```diff
@@ -174,7 +131,39 @@ class ProtobufService {
...
+      actionResponse = <ActionResponse>{
+        message: err.message,
+        success: false,
+      };
+    } else if (err instanceof Response) {
+      if (err.status.valueOf() == 403) {
+        // Not Found
+        actionResponse = <ActionResponse>{
+          message:
+            'Unauthorized access. The user lacks ___schema_manager permission.',
```
Comment (pruivo) [inline src/services/protobufService.ts:153]:
```suggestion
            'Unauthorized access.',
```

**Comment by karesti** [comment]
@pruivo let me know how it feels

**Comment by karesti** [comment]
@pruivo I upgraded some config that changed in jenkins now the build is launched

**Comment by pruivo** [comment]
works fine here. waiting for CI
In the meantime, I've created the following issue: https://github.com/infinispan/infinispan-console/issues/102

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**